patches/tModLoader/Terraria.Localization.Content/es-ES.tModLoader.json

```
@@ -98,7 +98,7 @@
 
 		// Mod Info
 		"ModInfoHeader": "Info del mod: ",
-		"ModInfoVisitHomepage": "Visita la página Web de Mod para más información.",
+		"ModInfoVisitHomepage": "Visita la página web de mod para más información.",
```

```suggestion
		"ModInfoVisitHomepage": "Visita la página web de el mod para más información.",
```
or
```suggestion
		"ModInfoVisitHomepage": "Visita la página web del mod para más información.",
```
would probably be more appropriate IMHO